pages/openbsd/cal.md

```diff
@@ -0,0 +1,32 @@
+# cal
+
+> Prints calendar information, with the current day highlighted.
+> More information: <https://man.openbsd.org/cal>.
+
+- Display a calendar for the current month:
+
+`cal`
+
+- Display a calendar for a specific year (4 digits):
+
+`cal {{year}}`
+
+- Display a calendar for a specific month and year:
+
+`cal {{month}} {{year}}`
+
+- Display a calendar for the current [y]ear:
+
+`cal -y`
+
+- Display [j]ulian days (one-based, numbered from January 1):
```

Comment:
What does "one-based" mean?

Reply:
Maybe we could write "without leading zeroes" then?

Reply:
> I think like 1, 2, 3 etc instead of 01, 02, 03 etc?

Actually, it never leaves a leading zero, no matter what option we use. One-based probably means "starting from 1" (instead of starting from 0)